OracleUnifiedDirectorySM/dockerfiles/12.2.1.4.0/Dockerfile

```diff
@@ -0,0 +1,149 @@
+#
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at 
+# https://oss.oracle.com/licenses/upl.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle Unified Directory Service Manager (OUDSM)
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# See oud.download file 
+# Also see patches.download file in the patches directory
+
+# ------------------------------------------------------------
+# Pull base image
+# ---------------
+FROM oracle/fmw-infrastructure:12.2.1.4.0 as base
+
+# Maintainer
+# ----------
+LABEL maintainer="OUD Team"
```

Comment:
Please review the `./CONTRIBUTING.md` file for the preferred set of default `LABEL` entries that should be provided for each image. In particular, you should not abbreviate Oracle product names, so at least this should be changed to "Oracle Unified Directory team."

Reply:
Updated